website/docs/components/scroll-container.mdx

```diff
@@ -0,0 +1,7 @@
+import { StorybookComponent } from "@site/src/components/StorybookComponent";
+
+# Scroll Container
+
+<StorybookComponent implementations={["core", "angular", "react"]} />
+
+🚧 Documentatie volgt 🚧
```

Comment:
Volgens mij moeten we hier ook maar eens mee stoppen. Kunnen we niet ergens wat documentatie vandaan halen? Zijn er bruikbare teksten uit het ticket, issue of Slack te halen?